docker/postgres/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM postgres:12-alpine
+ARG BUILD_DEPS="git make gcc libc-dev clang llvm"
+
+RUN apk --no-cache add $BUILD_DEPS \
```

Comment:
If you want to be able to rebuild this without purging your files whenever the test_parser repository changes, add this before the run command:

```
ADD https://api.github.com/repos/r888888888/test_parser/git/refs/heads/master /tmp/test_parser_version.json
```